src/cache_cf.cc

```diff
@@ -2908,9 +2962,14 @@ dump_time_t(StoreEntry * entry, const char *name, time_t var)
 void
 parse_time_t(time_t * var)
 {
-    time_msec_t tval;
-    parseTimeLine(&tval, T_SECOND_STR, false);
-    *var = static_cast<time_t>(tval/1000);
+    static const auto maxTime = std::numeric_limits<time_t>::max();
```

Comment:
I wonder whether `static` is justified here. You are using it to decrease the amount of computations that happen every time parse_time_t() is parsed, right? If `std::numeric_limits<time_t>::max()` is reduced to a numeric constant by an optimizing compiler, then using `static` may actually _increase_ computational costs because the compiler would be afraid of removing that static variable. And if the computational costs are the same because the compiler removes that static variable, we still should not use `static` because it raises questions like this while going against the "less ink is better" principle.